app/components/candidate_interface/references_component.rb

```diff
@@ -9,7 +9,7 @@ def initialize(application_form:)
     end
 
     def references
-      application_form.application_references
+      application_form.application_references.creation_order
     end
   end
```

Comment:
I think you could change also app/services/support_interface/candidate_journey_tracker.rb
Line 155, 140

Reply:
Done, thanks!